scripts/suspend.py

```diff
@@ -111,19 +112,15 @@ def delete_instances(instances):
     log.info(f"deleted {len(done)} instances {to_hostlist(done.keys())}")
 
 
-def suspend_nodes(nodelist):
-    """suspend nodes in nodelist"""
-    nodes = nodelist
-    if not isinstance(nodes, list):
-        nodes = util.to_hostnames(nodes)
-
-    tpu_nodes = []
+def suspend_nodes(nodes: List[str]) -> None:
+    tpu_nodes, other_nodes = [], []
```

Comment:
nit: nodes or compute nodes instead of `other_nodes`

Reply:
I disagree, both `nodes` and `compute nodes` don't any clarity, but rather reduce it.